src/domain/snake.py

```diff
@@ -1,15 +1,36 @@
-from dataclasses import dataclass
+from dataclasses import dataclass, field
 
 from autologging import traced
 
+from src.domain.del_kace import DelKace
+
 
 @dataclass
 @traced
 class Snake:
     x: int
     y: int
-    velikost: int
+    dx: int
+    dy: int
+    del_kace: list = field(default_factory=list)
```

Comment:
Del kače ne potrebuješ v tem classu saj imaš že `deli` in te spremenljivke sploh ne uporabljaš v logiki tega classa.